aptos-move/move-examples/mint_nft/step1/sources/nft_step1.move

```diff
@@ -0,0 +1,144 @@
+        "sequence_number": 0,
+        "success": true,
+        "timestamp_us": 1669659103283876,
+        "version": 12735152,
+        "vm_status": "Executed successfully"
+      }
+    }
+    */
+/// - 2.c Check the module we just published on the Aptos Explorer.
+/// Go to https://explorer.aptoslabs.com/. For the example above, we used devnet, so we will choose `devnet` at the
+/// top right of the screen and paste the transaction_hash `0x576a2e9481e71b629335b98ea75c87d124e1b435e843e7a2ef8938ae21bebfa3`
+/// in the `Search Explorer` box. We can see the changes we made by publishing this module under the `Changes` tab.
+///
+/// 3. Check out the `mint_nft()` function below - we are not going to run a command to mint the NFT in this step, because
+/// this function right now asks for two signers and that's inpractical to do using CLI commands.
+/// In the next step of this tutorial, we will introduce a way to programmatically sign for transactions, so the module publisher
+/// doesn't need to manually sign transactions, and only need one signer (the nft receiver's signer) for the `mint_nft()` function.
+module mint_nft::nft_step1 {
+    use std::error;
+    use std::string;
+    use std::vector;
+
+    use aptos_token::token;
+    use std::signer;
+    use std::string::String;
+    use aptos_token::token::TokenDataId;
+
+    // This struct stores an NFT collection's relevant information
+    struct ModuleData has key {
+        token_data_id: TokenDataId,
+    }
+
+    /// Action not authorized because the signer is not the admin of this module
+    const ENOT_AUTHORIZED: u64 = 1;
+
+    /// `init_module` is automatically called when publishing the module.
+    /// In this function, we create an example NFT collection and an example token.
+    fun init_module(source_account: &signer) {
+        let collection_name = string::utf8(b"Collection name");
+        let description = string::utf8(b"Description");
+        let collection_uri = string::utf8(b"Collection uri");
+        let token_name = string::utf8(b"Token name");
+        let token_uri = string::utf8(b"Token uri");
+        let maximum_supply = 0;
+        let mutate_setting = vector<bool>[ false, false, false ];
+
+        // Create the nft collection.
+        token::create_collection(source_account, collection_name, description, collection_uri, maximum_supply, mutate_setting);
+
+        // Create a token data id to specify the token to be minted.
+        let token_data_id = token::create_tokendata(
+            source_account,
+            collection_name,
+            token_name,
+            string::utf8(b""),
+            0,
+            token_uri,
+            signer::address_of(source_account),
+            1,
+            0,
+            // we don't allow any mutation to the token
+            token::create_token_mutability_config(
+                &vector<bool>[ false, false, false, false, true ]
+            ),
+            vector::empty<String>(),
```

Comment:
let's add some example property here as example to demonstrate how to use it?

Reply:
yeah good point!